models/prompt_command_factory.py

```diff
@@ -22,7 +22,7 @@ def create_command(prompt_type: str) -> PromptCommand:
             return SampleResumeExampleCommand()
         elif prompt_type in ["faq title"]:
             return FAQTitleCommand()
-        elif prompt_type in ["faq content"]:
-            return FAQContentCommand()
+        elif prompt_type.lower().strip() in ["faq content", "example"]:
+            return BlogDesignCommandFactory().create_command(blog_name, prompt_type)
```

Comment:
no need to instantiate the class you can just do (statically):

`return BlogDesignCommandFactory.create_command(blog_name, prompt_type)`